uber/templates/groups/history.html

```diff
@@ -30,14 +30,36 @@
 {% endfor %}
 </table>
 
+    {% if group.stripe_transactions %}
```

Comment:
I'd delete this change in another commit on this branch. If this somehow gets merged and #2471 doesn't get merged, then `group.stripe_transactions` will blow up, right?